app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@LuD1161 that's what I'm saying ... Make a check and ensure the offset coming as a part of the date time objects is the same as the offset of the timezone field. Use the api libs, field validation implementation to do it so that proper error responses are sent